qa/src/test/java/io/camunda/migrator/qa/history/HistoryMigrationListSkippedTest.java

```diff
@@ -72,32 +68,23 @@ public void shouldListAllSkippedHistoricEntities(CapturedOutput output) {
     }
 
     private List<String> createTestProcessInstances() {
-        List<String> processInstanceIds = new ArrayList<>();
```

Comment:
most is indentation update from 4 to 2 spaces.

Reply:
💭 Shall we check with the team if someone has an incorrect setting here? I've seen this in a few files

Reply:
We can ask in tomorrow's sync.